### ISSUE TITLE
run plot_logscales.py cannot get the same picture as show in the gallery

### ISSUE THREAD

**Issue by leeyupeng119** [bendichter/brokenaxes#60]
I ran the plot_logscales.py , but the output picture was not the same with the demo.
and I  use an anaconda python environment with  matplotlib  version 3.3.2 .
![Snipaste_2021-01-29_10-53-22](https://user-images.githubusercontent.com/55687784/106225804-d65fa500-6220-11eb-90a7-86a48981169a.png)

**Comment by bendichter** [comment]
@leeyupeng119 yes, thank you for bringing this up. You are right. It looks like this might be a change in the behavior of matplotlib. I'm not sure how to fix it.
